filebeat/tests/integration/journald_test.go

```diff
@@ -103,23 +66,16 @@ func TestJournaldInputRunsAndRecoversFromJournalctlFailures(t *testing.T) {
 
 	pid := 0
 	fmt.Sscanf(logEntry.Message, "journalctl started with PID %d", &pid)
-
-	filebeat.WaitForLogs("Count: 003", 5*time.Second, "did not find the third event in published events")
+	filebeat.WaitPublishedEvents(5*time.Second, 3)
 
 	// Kill journalctl
 	if err := syscall.Kill(pid, syscall.SIGKILL); err != nil {
 		t.Fatalf("coluld not kill journalctl with PID %d: %s", pid, err)
 	}
 
-	go generateJournaldLogs(t, context.Background(), syslogID, 5)
+	generateJournaldLogs(t, syslogID, 5, 100)
```

Comment:
I assume this writes at least 4kb, right? The total size of the payload is not very clear without making calculations based on the logic of the function.

Reply:
That's from another test where the size is not important. It's 500 bytes: 5 lines x 100 characters = 500 bytes.